include/mapper.h

```diff
@@ -19,19 +19,35 @@
 #ifndef DOSBOX_MAPPER_H
 #define DOSBOX_MAPPER_H
 
+#include "dosbox.h"
+
 #include <string>
 #include <vector>
-#include "setup.h"
-#include "types.h"
 
-enum MapKeys {
-	MK_f1,MK_f2,MK_f3,MK_f4,MK_f5,MK_f6,MK_f7,MK_f8,MK_f9,MK_f10,MK_f11,MK_f12,
-	MK_return,MK_kpminus,MK_scrolllock,MK_printscreen,MK_pause,MK_home
```

Comment:
Nice cleanup - removes a 2nd layer of indirection and allow using SDL_Scancode as-is.

Reply:
Yeah, I needed either to extend it with more values or clean up. These were literally used only in a single function…